tests/contrib/kombu/test.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+import kombu
+from nose.tools import eq_
+
+from ddtrace import Pin
+from ddtrace.contrib.kombu import get_traced_kombu
+from ddtrace.contrib.kombu.patch import patch, unpatch
+from ..config import RABBITMQ_CONFIG
+from ...test_tracer import get_dummy_tracer
+
+
+def test_kombu_legacy():
+    results = []
+
+    def process_message(body, message):
+        results.append(body)
+        message.ack()
+
+    # ensure the old interface isn't broken, but doesn't trace
+    tracer = get_dummy_tracer()
+    TracedKombu = get_traced_kombu(tracer, "foo")
+    conn = TracedKombu("amqp://guest:guest@127.0.0.1:%s//" % (RABBITMQ_CONFIG['port'], ))
+    conn.connect()
+    task_queue = kombu.Queue('tasks', kombu.Exchange('tasks'), routing_key='tasks')
+    producer = kombu.Producer(conn)
+    to_publish = {'hello': 'world'}
+    producer.publish(to_publish,
+                     exchange=task_queue.exchange,
+                     routing_key=task_queue.routing_key,
+                     declare=[task_queue])
+    with kombu.Consumer(conn, [task_queue], accept=['json'], callbacks=[process_message]):
+        conn.drain_events(timeout=2)
+
+    eq_(results[0], to_publish)
+    assert not tracer.writer.pop()
```

Comment:
We should check these spans more thoroughly as done below in the `_assert_conn_traced` function.